example/rebar.config

```diff
@@ -4,22 +4,42 @@
 %% == Erlang Compiler ==
```

Comment:
I ran `rebar3 format` over the example. These changes are probably best seen commit by commit.